lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -251,6 +256,134 @@ div.course-wrapper {
         }
       }
     }
+    .proctored-exam-skip-confirm-wrapper {
+      padding: $baseline 30px;
+      background-color: rgb(242, 244, 245);
+      border-left: 5px solid $red;
+      .msg-title {
+        margin-bottom: 12px;
+        font-size: 22px;
+        font-weight: 600;
+      }
+      .msg-content {
+        color: rgb(121, 118, 118);
+        strong {
+          font-weight: 600;
+        }
+      }
+      .proctored-exam-skip-actions {
+        margin-top: $baseline;
+        @include float(right);
```

Comment:
NIT: property order

We prefer to place positioning properties first, so let's move the `@include` above `margin-top` here.